packages/integration-tests/index.js

```diff
@@ -253,51 +264,126 @@ function setProcessEnvironment(stackName, bucketName) {
   process.env.stackName = stackName;
   process.env.messageConsumer = `${stackName}-messageConsumer`;
   process.env.KinesisInboundEventLogger = `${stackName}-KinesisInboundEventLogger`;
-  process.env.CollectionsTable = `${stackName}-CollectionsTable`;
```

Comment:
Is the idea that individual tests will set this up as needed? And that it unnecessarily couples table names to the stack names?

Reply:
The `setProcessEnvironment` function is only called from inside of the `integration-tests/index.js` file, so no test files were invoking it. After taking a closer look at it, the entire function can actually be deleted. I'll fix that.